docs/src/app/components/pages/components/grid-list.jsx

```diff
@@ -87,19 +87,19 @@ class GridListPage extends React.Component {
           },
           {
             name: 'actionPosition',
-            type: '"left"|"right"',
+            type: 'oneOf [left, right]',
```

Comment:
I think it'd be a better idea to retain the quotes around the values:

`oneOf ["left", "right"]`